media/stump-radio/radio.lisp

```diff
@@ -4,8 +4,8 @@
 
 (defparameter *stations*
```

Comment:
This should be left empty, the README is enough of an example.

Reply:
Then, by default, this small program would not do anything. There should be some default stations. I do not want to publish a program that does nothing by default.

But considering your comment I've added (and documented) a new function CLEAR-STATIONS to get rid of the defaults and restart with an empty list of stations.

Reply:
I think that by default it shouldn't do anything. Here are some programs that "do nothing" by default:
- A web browser
- A news reader
- An email client
... and I think this fits in that category also.

Reply:
StumpWM contrib modules' behavior is at the discretion of the author.

Reply:
Thanks